File: Selenium/Activity19.java
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity19 {
    public static void main(String[] args) {
        
        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/alerts");
        System.out.println("Page title: " + driver.getTitle());

        driver.findElement(By.id("confirmation")).click();

        
        Alert confirmAlert = driver.switchTo().alert();
        System.out.println("Text in alert: " + confirmAlert.getText());

        confirmAlert.accept();
        System.out.println(driver.findElement(By.id("result")).getText());

        driver.findElement(By.id("confirmation")).click();
        confirmAlert = driver.switchTo().alert();
        confirmAlert.dismiss();
        System.out.println(driver.findElement(By.id("result")).getText());
        driver.quit();
    }
}

File: Selenium/Activity19.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait

with webdriver.Firefox() as driver:
    wait = WebDriverWait(driver, timeout=10)
    driver.get("https://training-support.net/webelements/alerts")
    print("Page title is: ", driver.title)

    driver.find_element(By.ID, "confirmation").click()

    confirmAlert = wait.until(EC.alert_is_present())
    print("Text in alert: " + confirmAlert.text)
    confirmAlert.accept()
    print(driver.find_element(By.ID, "result").text)
    
    driver.find_element(By.ID, "confirmation").click()
    confirmAlert.dismiss()
    print(driver.find_element(By.ID, "result").text)
    driver.quit()